### DISCUSSION TITLE
How to change the background/border/transparency of the completion menu?

### DISCUSSION THREAD

**Discussion by PCBismarck** [LazyVim/LazyVim#2360] [Q&A] [answered]
when i type a function, it always pop such a prompt, but it mix up with the background code, how to make it more obvious?

![complete](https://github.com/LazyVim/LazyVim/assets/93439675/0265634a-fa50-4a64-9480-a87842289610)

**Comment by dpetka2001** [comment, marked as answer]
For setting border check this [comment](https://github.com/LazyVim/LazyVim/discussions/419#discussioncomment-5802918). For background color, you would have to check the `nvim-cmp` docs for which highlight groups it uses to change the colors of those groups (haven't done that myself).

**Reply by PCBismarck** [reply]
Thanks, the comment help me a lot

**Comment by PCBismarck** [comment]
I found the param of the function `cmp.config.window.bordered()` from [this](https://github.com/hrsh7th/nvim-cmp/blob/main/lua/cmp/config/window.lua)
With this function, I finally figure out how to set the background color and its transparency
about the param of `vim.api.nvim_set_hl()`, can look at [this](https://neovim.io/doc/user/api.html#nvim_set_hl())
this is my final config:

```lua
  {
    "hrsh7th/nvim-cmp",
    opts = function(_, opts)
      vim.api.nvim_set_hl(0, "PopMenu", { bg = "#1F2335", blend = 0 })
      local cmp = require("cmp")
      local win_opt = {
        col_offset = 0,
        side_padding = 1,
        winhighlight = "Normal:PopMenu,FloatBorder:FloatBorder,CursorLine:Visual,Search:None",
      }
      opts.window = {
        completion = cmp.config.window.bordered(win_opt),
        documentation = cmp.config.window.bordered(win_opt),
      }
      opts.experimental = {
        ghost_text = true,
      }
    end,
  },
```
